apps/fluent-tester/src/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -9,156 +13,263 @@ const svgProps: SvgIconProps = {
+      />
+      <TextInput value={message} onChangeText={setMessage} style={styles.textInput} />
+      <View style={styles.notification}>
+        <Notification
+          variant={variant}
+          onPress={() => {
+            console.log('Notification tapped');
+          }}
+        >
+          {message}
+        </Notification>
+      </View>
+    </View>
   );
 };
 
-const PrimaryOutlineBarTest: React.FunctionComponent = () => {
+const PrimaryWithAutoHide: React.FunctionComponent = () => {
+  const [visible, setVisible] = React.useState(false);
+  const onButtonPress = () => setVisible(!visible);
+
+  const [hidden, setHidden] = React.useState<boolean>(!visible);
+  const height = React.useRef(new Animated.Value(0)).current;
+
+  const show = () => {
+    setHidden(false);
+    Animated.sequence([
+      Animated.timing(height, {
+        toValue: -50,
+        duration: NOTIFICATION_SHOW_DURATION,
+        useNativeDriver: USE_NATIVE_DRIVER_IOS,
+        easing: Easing.elastic(NOTIFICATION_BOUNCINESS),
+      }),
+      Animated.delay(3000),
+      Animated.timing(height, {
+        toValue: 0,
+        duration: HIDE_DURATION,
+        useNativeDriver: USE_NATIVE_DRIVER_IOS,
+      }),
+    ]).start(() => {
+      setHidden(true);
+      setVisible(false);
+    });
+  };
+
+  const hide = () => {
+    Animated.timing(height, {
+      toValue: 0,
+      duration: HIDE_DURATION,
+      useNativeDriver: USE_NATIVE_DRIVER_IOS,
+    }).start(() => {
+      setHidden(true);
+    });
+  };
+
+  React.useLayoutEffect(() => {
+    if (visible) {
+      show();
+    } else {
+      hide();
+    }
+  }, [visible, height]);
+
+  const animatedViewProps = {
+    transform: [{ translateY: height }],
+  };
+
   return (
-    <Notification
-      variant={'primaryOutlineBar'}
-      onPress={() => {
-        console.log('Notification tapped');
-      }}
-    >
-      Mail Sent
-    </Notification>
+    <View>
+      <View style={styles.button}>
+        <Button onPress={onButtonPress} title={visible ? 'Hide' : 'Show'} />
+      </View>
+      {!hidden && (
+        <Animated.View style={[animatedViewProps]}>
+          <Notification
+            variant={'primary'}
+            action="Undo"
+            onPress={() => {
+              console.log('Notification tapped');
+            }}
+            onActionPress={() => {
+              console.log('Undo tapped');
+            }}
+          >
+            Mail Archived
+          </Notification>
+        </Animated.View>
+      )}
+    </View>
   );
 };
+const PrimaryBarWithAutoHide: React.FunctionComponent = () => {
+  const [visible, setVisible] = React.useState(false);
+  const onButtonPress = () => setVisible(!visible);
+
+  const [hidden, setHidden] = React.useState<boolean>(!visible);
+  const height = React.useRef(new Animated.Value(0)).current;
+
+  const show = () => {
+    setHidden(false);
+    Animated.sequence([
```

Comment:
Is there any difference with the bar animation code?

Reply:
Besides the show duration and lack of easing, no